### ISSUE TITLE
Add Dapr to PATH when using the MSI installer

### ISSUE THREAD

**Issue by shubham1172** [dapr/cli#1077]
## Describe the proposal

User can install the Dapr CLI (dapr.exe) in a custom path using the MSI installer. This proposal is to use the installer to also update the PATH variable and point to the Dapr CLI installation.

Documentation reference: https://wixtoolset.org/documentation/manual/v3/xsd/wix/environment.html

Something like this can be used to append the new directory to $env:PATH.
```xml
<Environment Action="set" Name="PATH" System="yes" Id="dapr_path" Part="last" Value="$(var.DaprEXEDirectory)" />
```

## Release Note

RELEASE NOTE: **UPDATE** MSI installer to update the path after Dapr CLI installation

**Comment by mukundansundar** [comment]
For now can be mitigated with a docs line to add installed cli directory to path

**Comment by shubham1172** [comment]
Closed by #1081
